invenio_requests/services/events/service.py

```diff
@@ -120,7 +139,7 @@ def create(
             uow=uow,
         )
 
-        # Persist record (DB and index)
+        # Persist record (DB and index) - top-level events only
```

Comment:
```suggestion
        # Persist record (DB and index)
```

Not the case anymore